app/controllers/admin/schools/cohorts/change_programme_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    module Cohorts
+      class ChangeProgrammeController < ::Admin::BaseController
+        skip_after_action :verify_authorized
+        skip_after_action :verify_policy_scoped
+        before_action :set_school_and_cohort
+        before_action :set_induction_choice_form, only: %i[confirm update]
+
+        def show
+          @induction_choice_form = InductionChoiceForm.new
+          @induction_choice_form.cohort = @cohort
+        end
+
+        def confirm
+          render :show unless @induction_choice_form.valid?
+        end
+
+        def update
+          ChangeInductionService.new(school: @school, cohort: @cohort).change_induction_provision(@induction_choice_form.programme_choice)
+
+          set_success_message heading: "Induction programme has been changed"
+          redirect_to admin_school_cohorts_path
+        end
+
+      private
+
+        def set_induction_choice_form
+          @induction_choice_form = InductionChoiceForm.new(params.require(:induction_choice_form).permit(:programme_choice))
+          @induction_choice_form.cohort = @cohort
```

Comment:
Do we want to set school here too?

Reply:
We could probably set the form before each action, if we handled empty params. Up to you whether you prefer it :) 

```
    def change_programme_params
      return {} unless params.key?(: induction_choice_form)

      params.require(: induction_choice_form).permit(: programme_choice)
    end
```

Reply:
It doesn't have or need a school. I think I prefer this way, since we keep the require on the param